### ISSUE TITLE
Some items don't add to your list immediately when you click them

### ISSUE THREAD

**Issue by sarahpearson** [creativecommons/list#85]
When you click a new item, it often looks like nothing happens. Then when you click another item, the first one shows up on your list along with the second one.

**Comment by billymeinke** [comment]
+1 Had this issue too, except that the first item I clicked never showed up in the list at all. Second and third ones did.

Web app, Firefox 34.0.5

**Comment by mattl** [comment]
Yeah, I've noticed this too. Stupid British people and their crappy code.

**Comment by mattl** [comment]
So, this is a result of two different things. Lack of variety of items, but also caching. Going to move this to the next release.
